modules/Microsoft.Web/sites/deploy.bicep

```
@@ -192,6 +192,9 @@ param hyperV bool = false
 ])
 param redundancyMode string = 'None'
 
+@description('Optional. The site publishing credential policy names which are associated with the sites.')
+param sitePublishingCredPolicyNames array = []
```

I'd suggest to treat this as an array of objects (same as `slots`) and to call it after the child resource

```suggestion
param basicPublishingCredentialsPolicies array = []
```